src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -565,7 +565,12 @@ public static bool IsOrContainsAccessibleAttribute(this ISymbol symbol, ISymbol
                 return true;
             }
 
-            return namespaceOrType.GetMembers().Any(nt => nt.IsOrContainsAccessibleAttribute(withinType, withinAssembly));
+            foreach (var nt in namespaceOrType.GetTypeMembers())
```

Comment:
If getting rid of the lambda is important, then a comment to prevent someone from putting it back might be nice.